lib/jsdom/browser/Window.js

```diff
@@ -67,6 +69,7 @@ function Window(options) {
   this._globalProxy = this;
 
   this.__timers = Object.create(null);
+  this.__animationFrameCallbacks = Object.create(null);
```

Comment:
Does this have to go on the Window object? Can it not be just a variable?

Reply:
I just copied `__timers`. Should I change both?

Reply:
I'm guessing there is a reason why `__timers` is still there. @Sebmaster/@domenic?

Reply:
It's used (or should be used) to stop all timers on window close, I think?

Reply:
Yeah but if we bring `stopAllTimers()` function into the Window constructor, it will have access to these variables as well w/o leaking them to the outside environment. If there are no more reasons, I think that should be done.

Reply:
Well, I'm not sure it's a great idea to add that closure (and all the other closures that currently reference window.__timers) into every window instance. I think the current setup is OK-ish.

Reply:
@domenic Right now, that function is the only one that uses `__timers` that is not already under the Window constructor. It would also be a move towards consistency to move it under the Window constructor as well.

Reply:
I pushed a commit which removed both from `this`. PTAL 🙂